### PR TITLE
Enhance/correct some examples in classes.md

### PR DESCRIPTION
**Pull request by RexJaeschke**
As a result of using the test harness, I made the following edits:

- used `System` namespace prefixes (later, removed these)
- replaced all en-dashes with hyphens
- added a wrapper class
- changed instance method to static
- removed backticks and other spurious characters from inside examples and console output
- added accessors
- fixed escape sequences

### CODE REVIEW

File: standard/classes.md
Comment (jskeet) [inline standard/classes.md]: I definitely want to hold off on this until we've had a discussion about it.

File: standard/classes.md
Comment (jskeet) [inline standard/classes.md]: Might be worth using a verbatim string literal here? `SplitPath(@"c:\Windows\System\hello.txt", out dir, out name);`

File: standard/classes.md
Comment (jskeet) [inline standard/classes.md]: Add space around the second ellipsis for consistency?

File: standard/classes.md
Comment (jskeet) [inline standard/classes.md]: It's slightly annoying that we can't just keep using ... for this. We may be able to add some "advanced configuration" to make it work... I'm fine with doing it like this for now, so long as we're consistent so we can find everywhere that needs it, and revert later on.

**Review by jskeet — COMMENTED** [review]
Glad to see this is finding issues. I'd like to hold off merging this PR until we've discussed the style convention of namespaces though.

File: standard/classes.md
Comment (RexJaeschke) [reply standard/classes.md]: Also remove all `/*` and `*/`, as they are added by the extractor.

File: standard/classes.md
Comment (RexJaeschke) [reply standard/classes.md]: I think we can here. I just failed to remove the comment delimiters when pasting the source back in from the post-extraction source

**Review by jskeet — COMMENTED** [review]
Looks good other than the points for discussion.

File: standard/classes.md
Comment (RexJaeschke) [reply standard/classes.md]: removed qualifier and marked "Incomplete"

**Comment by RexJaeschke** [comment]
@jskeet ready for another look by you

**Comment by RexJaeschke** [comment]
@jskeet @BillWagner I see in the TG2 meeting minutes from 2022-11-02, "We decided to include in the appropriate extraction templates a set of three or four commonly used namespaces (such as System, System.Collections, System.Collections.Generic, and System.Io), and to not use fully qualified names unless they are needed to support discussions on the use of such names."

I just searched the md files for explicit `using` namespace directives, and found the following:

```csharp
using System;
using System.Collections.Generic;
using System.Collections;
using System.Diagnostics;
using System.IO;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Runtime.InteropServices;
using System.Security.Permissions;
using System.Text;
using System.Threading;
```

I added this list to the top of (my private version of) the templates standalone-console, standalone-lib, and code-in-main. I then tested a few examples that needed one or more of these.

Now when these templates are used with an example that already explicitly has one or more of these directives, I get

> `warning CS0105: The using directive for '<namespace name>' appeared previously in this namespace`

If we use this approach, we can look at all the examples having explicit `using` directives, and if those are *not necessary* for explaining the example, we can remove them; otherwise, we leave them in and add `ignoredWarnings:["CS0105"]` to the metadata. And, of course, we remove *all* fully qualified prefixes from examples where they are not necessary for explaining the example.

How does that sound?

**Comment by jskeet** [comment]
Assuming I've followed you correctly while cooking dinner, that sounds fine.

**Comment by RexJaeschke** [comment]
@jskeet This is ready for review. Once this has been merged, I'll go over the other md files and remove the using <namespace> directives and explicit namespace prefixes, as well.

**Review by jskeet — APPROVED** [review]

**Pull request merged**
